Assets/Scripts/SanitySystem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SanitySystem : MonoBehaviour {
+
+    public float sanity;
+    private bool inSafeZone;
+
+	// Use this for initialization
+	void Start () {
+        sanity = 20.0f;
+        inSafeZone = true;
+	}
+	
+	// Update is called once per frame
+	void Update () {
+        if (!inSafeZone && sanity > 0.0f)
```

Comment:
sanity > 0.0f
is this check necessary, since we would likely perform some action (kill player) when his sanity reaches 0